### ISSUE TITLE
vg construct "-t" option doesn't work

### ISSUE THREAD

**Issue by verne91** [vgteam/vg#2412]
I am trying to construct the graph with multiple threads. But it seems to just use one CPU.

**Comment by faithokamoto** [comment]
Yeah, changing this would be a larger project; it currently works as intended.
